repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/atomfeed/FeedCacheInvalidator.java

```diff
@@ -77,15 +78,20 @@ public void setFeeds(List<AuthFeedReader> feeds) {
 
     @Override
     public void run() {
+
+        // Generate trans-id here so it is the same between multiple pages
+        String traceID = UUID.randomUUID().toString();
         while (!done) {
```

Comment:
you didn't add it to the slf4j context, without that it won't show up in our logs